runtime/js/40_testing.js

```diff
@@ -639,6 +639,7 @@
     optionsOrFn,
     maybeFn,
   ) {
+    core.opSync("op_bench_check_unstable");
```

Comment:
This information is already provided to 99_main.js: https://github.com/denoland/deno/pull/13400/files#diff-00a8b60a7dc00e07ca154218d7c1f478aedfa44a0b6055e558c1463ca1515dbb -- maybe it would be easier to just do something like this instead of introducing another op?

Reply:
I don't agree, seems like adding an op is less code than wiring it up through `99_main.js`